src/coreclr/dlls/mscoree/mscorwks_ntdef.src

```diff
@@ -274,6 +276,7 @@ EXPORTS
         mono_unity_class_try_get_vtable
         mono_unity_domain_mempool_chunk_foreach
         mono_unity_domain_set_config
+        mono_unity_domain_unload
```

Comment:
This should not be there. We don't want to unload anything yet (this will be done from C# when we work on the editor loader)